apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -208,6 +208,23 @@ describe('BillingReceipt', () => {
             expect(obj.id).toEqual(objUpdated.id)
             expect(objUpdated.period).toEqual('2011-12-01')
         })
+
+        test('toPay is Decimal', async () => {
+            const { organization, integration, managerUserClient } = await makeOrganizationIntegrationManager()
+            const [context] = await createTestBillingIntegrationOrganizationContext(managerUserClient, organization, integration)
+            const [property] = await createTestBillingProperty(managerUserClient, context)
+            const [billingAccount] = await createTestBillingAccount(managerUserClient, context, property)
+            const [obj] = await createTestBillingReceipt(managerUserClient, context, property, billingAccount)
+
+            const payload = {
+                toPay: '22.92',
+            }
+
+            const [objUpdated] = await updateTestBillingReceipt(managerUserClient, obj.id, payload)
+
+            expect(obj.id).toEqual(objUpdated.id)
+            expect(objUpdated.toPay).toEqual('22.92000000')
```

Comment:
Is that correct way to check decimals? at least it's not obvious for me

Reply:
Well, it looks odd, yep

The idea here is that you save something like 0.23 (note that it has 2 digits on decimal part) and DB parses it into 0.23000000 (note that it has 8 digits on decimal part as specified)